.gitignore

```diff
@@ -1,3 +1,6 @@
+# Ignore test data files
+**/connectedk8s/azext_connectedk8s/tests/latest/data/
+
```

Comment:
Why do you need to modify this file?

Reply:
Otherwise, you will commit a bunch of test files if you run the Azure Cloud tests.

Reply:
Those files I mentioned are autogenerated intermediate files from the test and were not committed before this PR, so I didn't see a reason to commit it for this PR.

Reply:
OK, got it. Please add this line to the end of the file instead of the beginning. 
> # Ignore test data files
In addition, please provide a more detailed explanation in the comment

Reply:
I can move it to the end of the file, but is a more detailed description required? I feel like you're supposed to have generalized descriptions so other teams can put their own test data under the same gitignore comment.

Reply:
In fact, the files in the `xxx/tests/latest/data/` directory of other module are useful files that will be used for testing. Therefore, the ignore logic is unique for `connectedk8s` module at present, so I suggest making a more detailed explanation

Reply:
Okay, it seems the tests are not supposed to be using tests/latest/data to dump one-time-use test output. I will remove the git ignore.